assets/js/components/KeyMetrics/MetricTileNumeric.js

```diff
@@ -43,19 +44,20 @@ export default function MetricTileNumeric( props ) {
 		subText,
 		previousValue,
 		currentValue,
+		error,
 	} = props;
 
 	const formatOptions = expandNumFmtOptions( metricValueFormat );
 
-	return (
+	return ! error ? (
 		<Widget noPadding>
 			<div className="googlesitekit-km-widget-tile googlesitekit-km-widget-tile--numeric">
 				<h3 className="googlesitekit-km-widget-tile__title">
 					{ title }
 				</h3>
 				<div className="googlesitekit-km-widget-tile__body">
 					{ loading && <PreviewBlock width="100%" height="68px" /> }
-					{ ! loading && (
+					{ ! loading && ! error && (
```

Comment:
This was probably left in by mistake, there's no need to check for `! error` here as the conditional render means it's always going to be true at this point...